Lib/typing.py

```diff
@@ -2119,6 +2119,9 @@ def __new__(cls, name, bases, ns, total=True):
             tp_dict.__total__ = total
         return tp_dict
 
+    def __init__(cls, name, bases, ns, total=True):
+        super().__init__(name, bases, ns)
+
```

Comment:
add an `__init__` to prevent the `total` argument from being passed to `__init_subclass__`